frontend/app/src/components/home/groupdetails/InviteUsersWithLink.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+    import { getContext } from "svelte";
+    import ShareIcon from "svelte-material-icons/ShareVariant.svelte";
+    import QR from "svelte-qr";
+    import CopyIcon from "svelte-material-icons/ContentCopy.svelte";
+    import { _ } from "svelte-i18n";
+    import Link from "../../Link.svelte";
+    import { iconSize } from "../../../stores/iconSize";
+    import { toastStore } from "../../../stores/toast";
+    import type { OpenChat, GroupChatSummary } from "openchat-client";
+    import { canShare, shareLink } from "../../../utils/share";
+
+    export let group: GroupChatSummary;
+
+    const client = getContext<OpenChat>("client");
+
+    $: link = `${window.location.origin}/${group.chatId}/?ref=${client.user.userId}`;
+
+    function onCopy() {
+        navigator.clipboard.writeText(link).then(
+            () => {
+                toastStore.showSuccessToast("linkCopiedToClipboard");
+            },
+            () => {
+                toastStore.showFailureToast("failedToCopyLinkToClipboard");
+            }
+        );
+    }
+
+    function onShare() {
+        shareLink(link);
+    }
+</script>
+
+<div class="link-enabled">
+    <div class="link">{link}</div>
+    <div class="qr-wrapper">
+        <div class="qr">
+            <QR text={link} />
+        </div>
+    </div>
+    <div class="message">
+        {$_("group.invite.shareMessage") +
```

Comment:
These two messages don't exist in the en.json file

Reply:
Think this must have got broken after a merge conflict